src/librados/IoCtxImpl.cc

```diff
@@ -52,10 +52,11 @@ struct CB_notify_Finish {
       preply_buf_len(_preply_buf_len) {}
 
 
+  // move-only
   CB_notify_Finish(const CB_notify_Finish&) = delete;
-  CB_notify_Finish operator =(const CB_notify_Finish&) = delete;
-  CB_notify_Finish(CB_notify_Finish&&) = delete;
-  CB_notify_Finish operator =(CB_notify_Finish&&) = delete;
+  CB_notify_Finish& operator =(const CB_notify_Finish&) = delete;
```

Comment:
This looks good. When I did this originally I was wanting to avoid moving away from an object someone had an oustanding pointer to, but I think I must have refactored the pointer I was worried about leaving dangling away.